### ISSUE TITLE
Provision to download specific version of XUnit Console runner  

### ISSUE THREAD

**Issue by vrohit13** [JetBrains/meta-runner-power-pack#96]
Should we introduce a new parameter to specify the specific version of XUnit version to be downloaded internally by the plugin on TeamCity Server. The main idea is not to use the latest version of XUnit as I have observed multiple issues across XUnit and .NetFramework. Restricting to a specific version will make sure that no issues come up.

So, NuGet seems to provide an option to download the specific version of Console Runner.
<NugetExe> install xunit.runner.console -source <XUnitSource> **-version 2.1.0**

Perhaps this new parameter could be an **OPTIONAL** parameter, which if empty would imply that the latest version should be fetched from NuGet.

So,
If Parameter value exists
<NugetExe> install xunit.runner.console -source <XUnitSource> -version <Version>

If Parameter value is empty,
<NugetExe> install xunit.runner.console -source <XUnitSource>

**Comment by dbckr** [comment]
I think this is a good idea. 2.3.0 released a few days ago and our XUnit build step started failing. Took a while to figure out it broke because it was trying to use a new version of the runner. Got it working again by adding `-version 2.2.0` to the meta-runner script.

**Comment by vrohit13** [comment]
Sure. I faced this issue 2 days back, and the workaround (based on the release notes from XUnit 2.3.0) is to modify the value against 'Xunit Runner Executable' in the Build Configuration.
![teamcity_xunit_2 3fix](https://user-images.githubusercontent.com/5303494/31456520-eab9966e-ae6e-11e7-8483-f8800622ecba.png)

**Comment by andrey3diq** [comment]
It would be nice to have a fix that will make it work with any xUnit version.
I like the idea suggested by @vrohit13 with optional Version parameter.

**Comment by dlbuller** [comment]
Agree, this would be good to have.  I can't seem to get my project to build correctly.

**Comment by djflash5** [comment]
I used the above fix 2 weeks ago and it saved the day but of course this morning with the release of 2.3.1 I'm seeing the same failure. The source package still has the .exe inside the net452 directory so I can't see why it would have broken. Anyone have any ideas?

**Comment by djflash5** [comment]
Strangely once I purged my Temcity buildagent's work directories everything is back to normal.
